plugins/auth-backend/src/providers/oidc/provider.ts

```diff
@@ -224,68 +31,23 @@ export class OidcAuthProvider implements OAuthHandlers {
  */
 export const oidc = createAuthProviderIntegration({
```

Comment:
Resolvers need to stay intact here as well, so that this is not a breaking change

Reply:
Sure thing will add the previous resolvers